src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +132,98 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
                                                         year=event_time.year)
     base_filter = QueryString('')
     transition_count = get_group_counts(transitioned_pop, base_filter, base_key, config, age_bins)
-    return transition_count
\ No newline at end of file
+    return transition_count
+
+
+
+class LiveBirthWithNTDObserver:
+    """Observes births and births with neural tube defects. Output can be stratified
+    by year and by sex.
+    """
+    configuration_defaults = {
+        'metrics': {
+            'disease_observer': {
+                'by_year': True,
+                'by_sex': True,
+            }
+        }
+    }
+
+    def __init__(self):
+        self.disease = project_globals.NTD_MODEL_NAME
+        self.configuration_defaults = {
+            'metrics': {f'{self.disease}_observer': LiveBirthWithNTDObserver.configuration_defaults['metrics'][
+                'disease_observer']}
+        }
+
+    @property
+    def name(self):
+        return f'live_birth_with_ntd_observer'
+
+    def setup(self, builder):
+        self.config = builder.configuration['metrics'][f'{self.disease}_observer'].to_dict()
+        self.live_birth_counts = Counter()
+        self.with_disease = Counter()
+        tmp = builder.configuration['time']['start']
+        self._reference_time = pd.Timestamp(tmp['year'], tmp['month'], tmp['day'])
```

Comment:
`reference_time` is ambiguous.  Try `self._simulation_start = pd.Timestamp(**builder.configuration.time.start.to_dict())`